portfolio/src/main/webapp/gallery.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Leonard Zhang's Photo Gallery</title>
+    <link rel="stylesheet" href="gallery_style.css">
+    <script src="change_picture_script.js"></script>
+  </head>
+  <body>
+    <p id="return-link"><a href="index.html"><< Go back to the main portfolio page</a></p>
+    <div id="content">
+      <h1>Welcome to my Photo Gallery!</h1>
+      <h2>Let's first see some pictures of mine.</h2>
+      <div class="image-thumbnails">
+        <a href="images/Leonard-1.jpg"><img src="images/Leonard-1.jpg"/></a>
+        <a href="images/Leonard-2.jpg"><img src="images/Leonard-2.jpg"/></a>
+        <a href="images/Leonard-3.jpg"><img src="images/Leonard-3.jpg"/></a>
+      </div>
+      <h2>I'm also a big fan of Gundam. I made a lot Gundam kits of the Mobile Suits in my spare time.</h2>
+      <div class="image-thumbnails">
+        <a href="images/Impulse Gundam.jpg"><img src="images/Impulse Gundam.jpg"/></a>
+        <a href="images/Nu Gundam.jpg"><img src="images/Nu Gundam.jpg"/></a>
+      </div>
+      <h2 id="hiking">When I go travelling with my family members, I love taking pictures of the amazing landscapes and scenarios.</h2>
+      <div id="button-container">
+        <button onclick="changeImage()" style="vertical-align:middle"><span>Click to see more photos!</span></button>
```

Comment:
Better to move this styling to the CSS file.

Reply:
I've deleted the button feature in gallery.html in the last commit. But I will remember to put stying to CSS file in the future.